tests/test_extend_schema.py

```diff
@@ -98,7 +98,22 @@ class DoesItAllViewset(viewsets.GenericViewSet):
                     enum=[True, False],
                     default=False,
                     description='creation will be in the sandbox',
-                )
+                ),
+                OpenApiParameter(
+                    name='X-Api-Version',
+                    type=str,
+                    location=OpenApiParameter.HEADER,
+                    response=True,
+                ),
+                OpenApiParameter(
+                    name='Location',
+                    type=OpenApiTypes.URI,
+                    location=OpenApiParameter.HEADER,
+                    description='URL of the created resource',
+                    response={
+                        201: True,
```

Comment:
will probably prefer `True` or a list like `['201', 200]`  as the dict value seems redundant

Reply:
yeah, was thinking the same. The difference between bool/dict (or list) doesn't sit really well with me but I don't see an obvious less-complicated solution either.

Reply:
yes, probably the least ugly solution. :smile: